kupfer/ui/style.css

```diff
@@ -0,0 +1,30 @@
+#kupfer {
```

Comment:
Super cool, but I will put in my caution: Css styling of GTK is messy, and it seems to change with every version of GTK. This default style file doesn't look like it defines enough colors and properties, so the style will be a mix of this file and the current theme, which can turn out very ugly.